### PR TITLE
Chef 16 fixes

### PR DESCRIPTION
**Pull request by daviddjh**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -6,9 +6,9 @@
 issues_url       'https://github.com/osuosl-cookbooks/dobc/issues'
 source_url       'https://github.com/osuosl-cookbooks/dobc'
 description      'Installs/Configures dobc'
-long_description 'Installs/Configures dobc'
 version          '2.0.4'
 
+depends          'base'
```
Comment (ramereth) [inline metadata.rb:11]: What is base needed for?

**Comment by daviddjh** [comment]
[Chef 16: dobc](https://trello.com/c/lgUKNAv1/2040-chef-16-dobc)

File: metadata.rb (hunk 1 above)
Comment (daviddjh) [reply metadata.rb:11]: Was getting `undefined method 'osl_shell_environment' for cookbook: osl-docker` without base defined in Berksfile. I think it was getting it off the supermarket.

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by ramereth** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.1.0.  Have a nice day!

**Pull request merged**